EventListener/IsGrantedListener.php

```diff
@@ -36,54 +41,95 @@ public function __construct(ArgumentNameConverter $argumentNameConverter, Author
         $this->authChecker = $authChecker;
     }
 
-    public function onKernelControllerArguments(KernelEvent $event)
+    public function onKernelController(FilterControllerEvent $event)
```

Comment:
same comment as on the other method, this type is deprecated, please check them all